packages/components/src/DataList/components/DataListFilters/components/DataListSort/DataListSort.tsx

```diff
@@ -13,12 +13,14 @@ export function DataListSort() {
 
   const sortByOptions = getSortByOptions();
 
+  const selectedSortID = `${state?.key},${state?.order},${state?.label},${state?.dataValue}`;
```

Comment:
ooh what happens if someone puts a comma on their label?

Reply:
much sad happens 😢 

![Screenshot 2024-01-26 at 5 32 33 PM](https://github.com/GetJobber/atlantis/assets/104797704/04cf9227-f7a7-4167-9e2b-9a20b4397a94)

It's also broken on master if you put a comma in there - the selection doesn't stick and we loose the full label 😭 

![Screenshot 2024-01-26 at 5 36 28 PM](https://github.com/GetJobber/atlantis/assets/104797704/8fd755ea-ded6-49f6-ae5f-218d0001b17b)

Reply:
i wonder if you can use an emoji 😆

Reply:
I could `JSON.stringify` everything and that does seem to fix it 🤔  Am I going in the right direction?

Reply:
could be. could also look at that sortID and see if the label needs to be there. You have an extra ID on it now, looks like you could just use the ID 🤔

Reply:
Okay I fixed the comma issue here c315056c2f8d04ffe2ef9a821c51a2504f623ddd 

I'm pretty sure label needs to be there so it goes back into the Combobox chip once a selection is made but if I'm misunderstanding what you're thinking, let me know.